tests/shared/src/test/scala/scala/meta/tests/parsers/ScaladocParserSuite.scala

```diff
@@ -516,14 +551,16 @@ class ScaladocParserSuite extends FunSuite {
                 "1.",
                 Seq(
                   ListItem(
-                    Text(Seq(Word(list11))),
-                    Some(
+                    Seq(
+                      Text(Seq(Word(list11))),
                       ListBlock(
                         "i.",
                         Seq(
                           ListItem(
-                            Text(Seq(Word(list21))),
-                            Some(ListBlock("I.", Seq(ListItem(Text(Seq(Word(list31)))))))
```

Comment:
if we go with my suggestion for `ListItem`, this diff becomes very small: `Some -> Seq`.